pkglistgen/update_repo_handler.py

```diff
@@ -1,6 +1,5 @@
 
 import glob
-import gzip
```

Comment:
There's other users of gzip module in this fule. See ci failure

Reply:
I did it differently, gzip and io are now gone